lucille-core/src/test/resources/MainIT/source.ndjson

```diff
@@ -0,0 +1,3 @@
+{"id":"3", "name":"Burger", "price":30, "country":"USA"}
```

Comment:
it looks like .jsonl is a more standard/common file extension for newline-delimited json than .ndjson
so let's rename this file to source.jsonl
there are a few other .ndjson files in the project that will need to be renamed but that can be done in a separate ticket

Reply:
@rseitz @akul-sethi FYI ticket to rename all .ndjson files to .jsonl is LC-476